packages/html-reporter/src/headerView.tsx

```diff
@@ -147,6 +134,18 @@ const SettingsButton: React.FC = () => {
       onMouseDown={preventDefault}>
       {icons.settings()}
     </a>
+    <Dialog
```

Comment:
It is important that `anchor` ref is placed in the DOM **before** the `Dialog`. Otherwise, React does not have it initialized by the time we measure the ref.